source/usage-examples/updateOne.txt

```diff
@@ -0,0 +1,32 @@
+=================
+Update a Document
+=================
+
+.. default-domain:: mongodb
+
+You can update a single document using the `updateOne() <https://mongodb.github.io/node-mongodb-native/3.3/api/Collection.html#updateOne>`_ method. 
+``updateOne()`` takes a filter for a document, and updates the first document
+that matches the filter.
+
+Create an Options object to specify additional options. Specify the
+``upsert`` option to create a new document if no documents match the
+filter.  To receive acknowledgment in the response from MongoDB that the
+write operation was performed on the specified number of :manual:`mongod
+</reference/program/mongod>` servers, set the ``w`` option. 
+
+The ``updateOne()`` method returns a document containing information such
```

Comment:
I wonder if it might be less confusing to use the term object here, even though this is a document, since using the same term to refer to the operation summary document and the items in the collection might be a little confusing for new users who don't have all the vocabulary down yet. Maybe an example document in a code snippet could help with this?

Also, I assume the method actually returns a Promise that resolves to the operation summary document since you use ``await`` in the code sample -- we should definitely mention that.

Reply:
Note: I'm making a new PR for this, but to address your comment:

I'm updating the new pr to specify that it returns a promise, and I've changed the wording to `object`

thanks